coalib/output/ConsoleInteraction.py

```diff
@@ -71,9 +71,7 @@ def highlight_text(no_color, text, style, lexer=TextLexer()):
 
 STR_GET_VAL_FOR_SETTING = ('Please enter a value for the setting \"{}\" ({}) '
                            'needed by {} for section \"{}\": ')
-STR_LINE_DOESNT_EXIST = ('The line belonging to the following result '
-                         'cannot be printed because it refers to a line '
-                         "that doesn't seem to exist in the given file.")
+STR_LINE_DOESNT_EXIST = 'Empty file'
```

Comment:
I don't think this would be the expected solution. Are you consulting @Makman2 on this? If not, I think you should.